vms/platformvm/api/camino.go

```diff
@@ -22,7 +22,13 @@ import (
 	"github.com/ava-labs/avalanchego/vms/types"
 )
 
-var errNonExistingOffer = errors.New("non existing deposit offer")
+var (
+	errNonExistingOffer             = errors.New("non existing deposit offer")
+	errWrongUTXONumber              = errors.New("wrong utxo number")
+	errWrongValidatorNumber         = errors.New("wrong validator number")
+	errWrongDepositsAndStakedNumber = errors.New("deposits and staked are not the same number")
```

Comment:
same, there we have not matching amount of dep definitions with validator staked outs